.gitignore

```diff
@@ -3,3 +3,6 @@ __pycache__
 env
 lmnop.sqlite
```

Comment:
the database lmnop.sqlite is still in your local git cache. You need to run `git rm -r --cached lmnop.sqlite`